src/Runner.Listener/Configuration/OAuthCredential.cs

```diff
@@ -36,7 +36,8 @@ public override VssCredentials GetVssCredentials(IHostContext context)
             // We expect the key to be in the machine store at this point. Configuration should have set all of
             // this up correctly so we can use the key to generate access tokens.
             var keyManager = context.GetService<IRSAKeyManager>();
-            var signingCredentials = VssSigningCredentials.Create(() => keyManager.GetKey());
+            var settings = context.GetService<IConfigurationStore>().GetSettings();
```

Comment:
if we store in the `credentialdata.data`, you can get it by
```
var requireFipsCryptography = StringUtil.ConvertToBoolean(this.CredentialData.Data.GetValueOrDefault("requireFipsCryptography"), false);
```
in line 32

Reply:
👍 This update has been made

Reply:
you can now remove `var settings = context.GetService<IConfigurationStore>().GetSettings();` i think.